build.gradle

```diff
@@ -1,5 +1,5 @@
 group 'uk.ac.ebi.subs'
-version '0.1.0-SNAPSHOT'
+version '0.2.0-SNAPSHOT'
```

Comment:
We don't need to update the version on the components that haven't been deployed to sonatype.